lms/djangoapps/certificates/queue.py

```diff
@@ -131,14 +131,8 @@ def regen_cert(self, student, course_id, course=None, forced_grade=None, templat
             certificate = GeneratedCertificate.eligible_certificates.get(user=student, course_id=course_id)
 
             LOGGER.info(
-                (
-                    "Found an existing certificate entry for student %s "
-                    "in course '%s' "
-                    "with status '%s' while regenerating certificates. "
-                ),
-                student.id,
-                str(course_id),
-                certificate.status
+                f"Found an existing certificate entry for student {student.id} in course '{course_id}' with status "
```

Comment:
Do we need _str(course_id)_, both here and below?

Reply:
My understanding of f-string formatting is that it automatically calls the underlying `__str__()` function when formatting. I was thinking I wouldn't need that `str(course_id)` function call anymore.

I tested this logging change in devstack and didn't run into any issues. The data came through in the logging as I would expect. Some examples:
`2021-04-06 14:21:13,043 INFO 6496 [lms.djangoapps.certificates.queue] [user 3] [ip 172.18.0.1] queue.py:132 - Found an existing certificate entry for student 3 in course 'course-v1:edX+Hynes101+1T2021' with status 'downloadable' while regenerating certificates.`
and
`2021-04-06 14:21:14,280 INFO 6496 [lms.djangoapps.certificates.queue] [user 3] [ip 172.18.0.1] queue.py:152 - The certificate status for student 3 in course 'course-v1:edX+Hynes101+1T2021' has been changed to 'unavailable'.`